templates/_includes/last_updated.html

```diff
@@ -7,7 +7,7 @@
 {% if delta > 0 %}
 <h4>Last Updated</h4>
 {% set day = article.modified.strftime('%d')|int %}
```

Comment:
It seems we do not need day variable after your patch. Can you please remove it in the same commit?

Reply:
Ah, I see, thanks for pointing that out. :+1:  Let me have a look and get back to this.

Reply:
done